recipes/journey_1/module_3/probe.yaml

```diff
@@ -197,15 +243,24 @@ tasks:
               content: "Instructions are a critical component of a well-constructed prompt as they provide explicit directions to the AI model on what actions to take or what kind of output is required. Clear and detailed instructions help the model understand the user's expectations and generate more accurate and relevant responses. Without specific instructions, the model might produce outputs that do not align with the user's needs."
         edges:
           - text: "Give me an example, Ada!"
+            target_node_id: screen4b
+          - text: "On to the next, Ada!"
             target_node_id: END
+      - id: screen4b
+        type: message
+        body:
+          parts:
+            - type: "text"
+              content: "For example, instead of just saying 'Create a script,' a prompt with clear instructions would be 'Create a Python script that connects to a MySQL database, retrieves all records from the 'users' table, and prints each user's name and email address.' These instructions guide the AI to focus on the specific requirements of the task."
+        edges:
           - text: "On to the next, Ada!"
             target_node_id: END
       - id: END
         type: message
         body:
           parts:
             - type: "text"
-              content: "For example, instead of just saying 'Create a script,' a prompt with clear instructions would be 'Create a Python script that connects to a MySQL database, retrieves all records from the 'users' table, and prints each user's name and email address.' These instructions guide the AI to focus on the specific requirements of the task."
+              content: "Here comes the next one, {username}! Let's explore the importance of context in prompt engineering."
```

Comment:
Duplication. When a new question is presented, there should only be one message in that node. Here, the END node has the same text as the next node before the question which will produce a duplication

![image](https://github.com/user-attachments/assets/bdcefb01-b000-43af-930f-f8f439748e4e)